src/ostorlab/serve_app/oxo.py

```diff
@@ -793,6 +794,60 @@ def mutate(
 
         runtime_instance: runtime.LocalRuntime = runtime.LocalRuntime()
         runtime_instance.follow = []
+        created_scan = runtime_instance.prepare_scan(
+            assets=scan_assets, title=scan.title
+        )
+        RunScanMutation._persist_scan_info(created_scan, scan)
+
+        executor = futures.ThreadPoolExecutor(max_workers=1)
+        executor.submit(
+            RunScanMutation._run_scan_background,
+            runtime_instance,
+            agent_group,
+            scan,
+            scan_assets,
+        )
+        return RunScanMutation(scan=created_scan)
+
+    @staticmethod
+    def _persist_scan_info(
```

Comment:
Why static methods?

Reply:
As discussed, these are specific to the mutation so they should be static